### PR TITLE
Feature replacesettingsscreen

### PR DESCRIPTION
**Pull request by Joltrix**
Removed the settings link button from the Launcher navigation menu (the left bar), and added the content of the settings to be displayed with the click of the "system button" (the top right cogwheel button)

### CODE REVIEW

**Comment by Joltrix** [comment]
Implemented #30

File: src/UserAdmin/wwwroot/UserAdmin/viewmodels/launcher/AppMenuPage.html [hunk 1]
```diff
@@ -14,11 +14,11 @@
                     <div title="System Users" class="the-icons span3"><i class="glyphicon glyphicon-user"></i> <span class="i-name">System User Groups</span></div>
                 </a>
             </li>
-            <li>
+            <!--<li>
```
Comment (warpech) [inline src/UserAdmin/wwwroot/UserAdmin/viewmodels/launcher/AppMenuPage.html]: Please avoid commenting-out code. Just remove the lines that are no longer needed

File: src/UserAdmin/Api/LauncherHooks.cs [hunk 2]
```diff
@@ -89,6 +89,7 @@ public class LauncherHooks {
             UriMapping.Map("/useradmin/menu", "/sc/mapping/menu");
             UriMapping.Map("/useradmin/app-name", "/sc/mapping/app-name");
             UriMapping.Map("/useradmin/search/{?}", "/sc/mapping/search?query={?}");
+            UriMapping.Map("/UserAdmin/admin/settings", UriMapping.MappingUriPrefix + "/settings");
```
Comment (warpech) [inline src/UserAdmin/Api/LauncherHooks.cs]: For code hygiene, could you also replace `"/sc/mapping` with `UriMapping.MappingUriPrefix + "`?

The code will look more consistent.

File: src/UserAdmin/MasterPage.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
     "Html": "",
-	"Menu": {},
+	  "Menu": {},
```
Comment (warpech) [inline src/UserAdmin/MasterPage.json]: Can you try to set up your Visual Studio to produce the desired JSON formatting? See [guidelines](https://github.com/StarcounterPrefabs/Guidelines/blob/master/coding_style/json_code_formatting.md)

**Comment by warpech** [comment]
Good job! I have commented about just a few minor things.

**Comment by Joltrix** [comment]
I fixed what you commented on, and commited them :)

**Comment by warpech** [comment]
Looking good! Can I merge?

**Comment by Joltrix** [comment]
Ah, good!

Yeah! Go ahead :)

**Comment by warpech** [comment]
Merged

**Pull request merged**